docs/plugins_configuration.rst

```diff
@@ -138,9 +153,9 @@ package:
 
    # File: MyPackage/interface.py
    import abc
-   from smqtk_core import Configurable, Pluggable
+   from smqtk_core import Plugfigurable
 
-   class MyInterface(Configurable, Pluggable):
+   class MyInterface(Plugfigurable):  # Transitively inherits from Pluggable and Configurable this is a really long line
```

Comment:
Can you move this comment to the line above and remove the self-snark.

Reply:
Oh wow, I didn't mean for that to get through. I was testing a setting in my IDE! :D Let me get rid of that...

Reply:
As in move the comment to right above the class definition like this?
```python
   # Transitively inherits from Pluggable and Configurable
   class MyInterface(Plugfigurable):
       ...
```

Reply:
That's what I meant but it should actually go wherever it's supposed to (but I'd suggest not to put it where it was, even without the "lengthener").

Reply:
I'm not sure where it "should actually go," so I made the comment into a doc-string for the example class.